src/com.meego.usb_moded.xml

```diff
@@ -82,6 +82,7 @@
     <signal name="sig_usb_taget_mode_config_ind">
       <arg name="config" type="a{sv}"/>
       <annotation name="org.qtproject.QtDBus.QtTypeName.In0" value="QVariantMap"/>
+      <annotation name="org.qtproject.QtDBus.QtTypeName.Out0" value="QVariantMap"/>
```

Comment:
I do not have qt6 tools readily available for testing, so can't be 100% sure about this but: as this deals with dbus signal, only "out" direction is applicable -> the "in" bit is some kind of unnoticed typo/thinko/stte -> replace "in" with "out" instead of having both (similarly to how the directly related "get_target_mode_config" method call already is).

Reply:
I am still not sure how to verify that everything works properly.

Reply:
I tested the original xml with qt5 tools and got the same "you should add annotation" warning. And the warning goes away when annotation name is changed from "In0" to "Out0" like you did -> the qt `<annotation>` bit is okay now.

But vanilla dbus things were already correct i.e. `<arg>` of `<signal>` has implied 'direction="out"` and does not need to be explicitly mentioned -> undo changes within `<arg>` tags.

Reply:
cosmetic issue. resolving discussion